src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisConfigDrawer.tsx

```diff
@@ -186,7 +189,7 @@ export const AutomatedAnalysisConfigDrawer: React.FC<AutomatedAnalysisConfigDraw
                 splitButtonVariant="action"
                 toggleVariant="default"
               >
-                Create Recording &nbsp;&nbsp;
+                {t('AutomatedAnalysisConfigDrawer.CREATE_RECORDING')} &nbsp;&nbsp;
```

Comment:
One idea here is that we can use margin with span so we can control how much space to add. Wdu think?

```typescript
<span style={{ marginRight: '0.5em'}}>
  {t('AutomatedAnalysisConfigDrawer.CREATE_RECORDING')}
</span>
```